core/src/test/java/org/apache/calcite/util/format/FormatElementEnumTest.java

```diff
@@ -77,8 +80,26 @@ class FormatElementEnumTest {
     assertFormatElement(FormatElementEnum.FF6, "2014-09-30T10:00:00.123456Z", "000123");
   }
 
-  @Test void testIW() {
-    assertFormatElement(FormatElementEnum.IW, "2014-09-30T10:00:00Z", "40");
+  @Test void testID() {
+    assertFormatElement(FormatElementEnum.ID, "2014-09-30T10:00:00Z", "2");
+  }
```

Comment:
I don't understand why some of the tests in this class are parameterized and others are not.

Frankly I don't think the framework is helping make the tests more understandable. I would move to code, and add comments if a test is testing something subtle.

It seems to me that this test class should be very simple.

Reply:
As there is one method for each format element, I didn't want to add new methods to test the edge cases of  ISOWEEK and ISOYEAR, and I thought a good solution was to make those tests parameterized.

But I agree that the way it was done didn't help in making the added test cases understandable. I removed the parameters and made more assertions inside the method (with a comment).